### DISCUSSION TITLE
Kali Linux not booting after installation

### DISCUSSION THREAD

**Discussion by theakshat1** [utmapp/UTM#2536] [UTM Help (macOS)]
I installed Kali Linux ARM64 on Mac m1 and it booted after installation but when I shut down it and closed the UTM the next time I am trying to boot VM it is asking me to install it again. I don't know what I am doing wrong

**Comment by 0xmra** [comment]
You might not have provided the system with the adequate space need for running the VM boot.
Try providing it at least 4-20 Gb it should work.

and if it doesn't then try reinstalling the UTM. 

<br>

![image](https://user-images.githubusercontent.com/52797899/120756347-c3414680-c52c-11eb-8387-a2023afd78cc.png)

![image](https://user-images.githubusercontent.com/52797899/120756176-90974e00-c52c-11eb-9574-c323f8dcf816.png)

**Reply by theakshat1** [reply]
I already did everything 4gb ram allocated and 10gb to vm , reinstalled everything 5 times but it didn't work

**Reply by alphadi** [reply]
I'm facing the same issue. Were you able to find a workaround?

**Reply by theakshat1** [reply]
Try https://github.com/utmapp/UTM/issues/4002#issue-1233405414

**Comment by 0xmra** [comment]
Try other VM's like Parallels Access or VirtualBox. Your system might not be compatible with running Linux with the UTM.
